subsys/bluetooth/controller/Kconfig

```
@@ -395,20 +395,41 @@ config BT_CTLR_SDC_EVENT_TRIGGER
 config BT_CTLR_SDC_ISO_RX_PDU_BUFFER_PER_STREAM_COUNT
 	int "BT_CTLR_SDC_ISO_RX_PDU_BUFFER_PER_STREAM_COUNT"
 	depends on BT_CTLR_SYNC_ISO || BT_CTLR_CONN_ISO
-	range 0 10
+	range 0 32
 	default 3
+	help
+	  Configures the number of RX PDU buffers allocated per ISO stream.
+	  This is the number of maximum size (251 bytes) PDU buffers.
+	  When PDU size is smaller than the maximum size, the pool will be repartitioned
+	  into a larger number of adjusted PDUs.
+	  For BIS, this value determines how many pretransmissions can be stored and utilized.
 
 config BT_CTLR_SDC_ISO_TX_HCI_BUFFER_COUNT
 	int "BT_CTLR_SDC_ISO_TX_HCI_BUFFER_COUNT"
 	depends on BT_CTLR_ADV_ISO || BT_CTLR_CONN_ISO
 	range 0 10
 	default 3
+	help
+	  Configures the number of shared HCI TX buffers allocated for ISO.
```

I would split this into two sentences for clarity.

```suggestion
	  Configures the number of HCI TX buffers allocated for ISO.
	  The HCI buffers are shared betwen the different streams.
```